microsoft-edge/devtools-guide-chromium/whats-new/2025/01/devtools-132.md

```diff
@@ -0,0 +1,88 @@
+---
+title: What's New in DevTools (Microsoft Edge 132)
+description: "CSS selector costs include links in Insights sidebar. Right-click a tab to move it left or right in Quick View. Heap snapshot objects categorized by constructor names and source locations in Memory tool. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 1/16/2025
+---
+# What's New in DevTools (Microsoft Edge 132)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## CSS selector costs now include links in the Insights sidebar
+
+<!-- Subtitle: Selectors have clickable links to their stylesheet in the Insights sidebar. -->
+
+The left sidebar in the **Performance** tool displays the costs of CSS selectors in the **Insights** tab when the **Enable CSS selector stats (slow)** checkbox is enabled.  Now, the selectors have links to the stylesheets that contain them.
+
+Some selectors might have more than one link when they are defined in multiple locations in a stylesheet or defined in multiple stylesheets.  For example, two links will be shown as **[1], [2]**. Selectors from user agent stylesheets or dynamically created stylesheets won't have links. 
+
+![CSS selector costs links](./devtools-132-images/selector-stats-links.png)
+
+See also:
+* [View CSS selector costs in the Performance tool sidebar](../../2024/11/devtools-131.md#view-css-selector-costs-in-the-performance-tool-sidebar) in _What's New in DevTools (Microsoft Edge 131)_.
+
+
+<!-- ====================================================================== -->
+## Move a tab left or right in the Quick View toolbar by right-clicking it
+
+<!-- Subtitle: Move the tabs in the Quick View toolbar left or right by using the tab's right-click menu. -->
+
+Previously, you could rearrange the tabs in **Quickk View** only by dragging them.  This can be difficult for users who are using input devices such as a trackball or head pointer.  Now, you can also move a tab left or right by right-clicking the tab and then selecting **Move left** or **Move right**:
```

Comment:
```suggestion
Previously, you could rearrange the tabs in **Quick View** only by dragging them.  This can be difficult for users who are using input devices such as a trackball or head pointer.  Now, you can also move a tab left or right by right-clicking the tab and then selecting **Move left** or **Move right**:
```